### PR TITLE
Adding slaac wpantund test

### PR DESCRIPTION
**Pull request by priyankaCh22**

### CODE REVIEW

File: silk/tests/openthread/ot_test_slaac_wpantund_test.py [hunk 1]
```diff
@@ -0,0 +1,326 @@
+# Copyright 2019 Google LLC
```
Comment (qilinkang) [inline silk/tests/openthread/ot_test_slaac_wpantund_test.py]: 2020

**Review by qilinkang — APPROVED** [review]
LGTM

**Comment by priyankaCh22** [comment]
@googlebot I signed it!

File: silk/tools/wpan_util.py [hunk 2]
```diff
@@ -37,9 +37,8 @@ def verify(condition):
         error_message = 'verify() failed at line {} in "{}"'.format(calling_frame.f_lineno,
                                                                     calling_frame.f_code.co_filename)
         if not _is_in_verify_within:
-            print error_message
-        return False
-    return True
+            print(error_message)
```
Comment (jenniexie) [inline silk/tools/wpan_util.py]: can we use logger?

Comment (priyankaCh22) [reply silk/tools/wpan_util.py]: Sure. Thanks for the suggestion. Have posted updated code using logger. 
Thanks

**Review by jenniexie — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Pull request merged**